### PR TITLE
Fix Grid variables not available for orphan <Row/>, see #7868

### PR DESCRIPTION
**Pull request by andrey-skl**

### CODE REVIEW

**Review by Hypnosphi — CHANGES_REQUESTED** [review]

File: src/grid/grid.css [hunk 1]
```diff
@@ -4,7 +4,8 @@
 @value large-screen-media, middle-screen-media, small-screen-media from "../global/global.css";
 
 .container-fluid,
-.container {
+.container
+.row {
```
[the remark this answers is not in the document]
Comment (Hypnosphi) [reply src/grid/grid.css:8]: Although I double-checked and we never use `Col` without having `Row` in the same component, so it should be fine as is

**Review by Hypnosphi — APPROVED** [review]

File: src/grid/grid.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrey-skl) [reply src/grid/grid.css:8]: Thanks for checking!

File: src/grid/grid.css [hunk 2]
```diff
@@ -4,7 +4,8 @@
 @value large-screen-media, middle-screen-media, small-screen-media from "../global/global.css";
 
 .container-fluid,
-.container {
+.container
```
Comment (Hypnosphi) [inline src/grid/grid.css:7]: a comma is missing here, so it's `.container .row` instead of `.container, .row`

Comment (Hypnosphi) [reply src/grid/grid.css:7]: I'll fix that

Comment (andrey-skl) [reply src/grid/grid.css:7]: Ouch I'm sorry, thank you for your help!

**Pull request merged**